### PR TITLE
Github template for reporting a documentation issue

### LINKED ISSUE #693: Meta Feature request: documentation issue template
### Use case  
template for reporting documentation issues.

### Is your feature request related to a problem?  
The two choices for reporting issues are:
* bug 
* feature request

### Describe your preferred solution 
button you can click "documentation problem"

page:

problem:

documentation should say:

### Describe any alternatives you have considered  
one giant issue for documentation

### PR DESCRIPTION
**Pull request by hkershaw-brown**
## Description:

An issue template for documentation problems. 

Also fixes a typo in the bug report template so the issue gets the label 'bug'

### Fixes issue

fixes #693 

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [x] GitHub feature

### Documentation changes needed?

- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly.

### Tests
Made an issue on my fork for docs: https://github.com/hkershaw-brown/DART/issues/5
Made an issue on my fork for a bug: https://github.com/hkershaw-brown/DART/issues/6 (bug label)

## Checklist for merging

- [ ] Updated changelog entry
- [ ] Documentation updated
- [ ] Update conf.py

## Checklist for release
- [ ] Merge into main
- [ ] Create release from the main branch with appropriate tag
- [ ] Delete feature-branch

## Testing Datasets

- [ ] Dataset needed for testing available upon request
- [ ] Dataset download instructions included
- [x] No dataset needed

### CODE REVIEW

**Review by mjs2369 — COMMENTED** [review]
https://github.com/NCAR/DART/blob/f193ab47f8079445f3a58c848a2b54469c130a84/.github/ISSUE_TEMPLATE/feature_request.md?plain=1#L5

We need to add the single quotation marks to this enhancement label for the Feature request issue template as well

File: .github/ISSUE_TEMPLATE/documentation_issue.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+labels: 'documentation'
+assignees: ''
+
+---
+
+### What's the issue? 
+:books: Please describe the problem you’ve noticed in the documentation. 
+
+### Where did you find the issue?
+Provide the location of the issue:
```
Comment (mjs2369) [inline .github/ISSUE_TEMPLATE/documentation_issue.md:14]:
```suggestion
Provide the location(s) of the issue:  
```

**Comment by hkershaw-brown** [comment]
> We need to add the single quotation marks to this enhancement label for the Feature request issue template as well

ENHANCE! nice spot eagle eyes. 🦅

File: .github/ISSUE_TEMPLATE/documentation_issue.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+Provide the location of the issue:  
+
+- File name or URL: 
+- Section name or header:
+
+### What needs to be fixed?
+Share what’s incorrect, unclear, missing, or outdated.
+
+### Suggestions for improvement
+If you have any ideas on how to fix the issue, please share them.
```
[the remark this answers is not in the document]
Comment (hkershaw-brown) [reply .github/ISSUE_TEMPLATE/documentation_issue.md:23]: yup this seems nicer

**Review by mjs2369 — APPROVED** [review]

**Pull request merged**